reactor-netty-http/src/main/java/reactor/netty/http/server/logging/AccessLogFactory.java

```diff
@@ -16,18 +16,53 @@
 package reactor.netty.http.server.logging;
 
 import java.util.function.Function;
+import java.util.function.Predicate;
 
-/**
- * An interface to declare more concisely a {@link Function} that apply an {@link AccessLog} by an
- * {@link AccessLogArgProvider}.
- * <p>
- * Can be used in {@link reactor.netty.http.server.HttpServer#accessLog(boolean, AccessLogFactory) accessLog} method
- * for example.
- *
+ /**
  * @author Simon Baslé
  * @author Audrey Neveu
  * @since 1.0.3
  */
 public interface AccessLogFactory extends Function<AccessLogArgProvider, AccessLog> {
 
-}
\ No newline at end of file
+	/**
+	 * Helper method to create an access log factory that selectively enables access logs.
+	 * <p>
+	 * Any request (represented as an {@link AccessLogArgProvider}) that doesn't match the
+	 * provided {@link Predicate} is excluded from the access log. Other requests are logged
+	 * using the default format.
+	 *
+	 * @param predicate the filter that returns {@code true} if the request should be logged, {@code false} otherwise
+	 * @return an access log factory {@link Function} to be used in
+	 * {@link reactor.netty.http.server.HttpServer#accessLog(boolean, AccessLogFactory)}
+	 * @since 1.0.3
+	 */
+	static AccessLogFactory create(Predicate<AccessLogArgProvider> predicate) {
```

Comment:
I still get a strong feeling the `create` keyword is too generic. An `AccessLogFactory` is not inherently filtering... wdyt @violetagg?

Reply:
@simonbasle @aneveu I will leave the decision to you